pkg/driver_sx126x/contrib/driver_sx126x_hal.c

```diff
@@ -33,6 +33,14 @@
 #define SX126X_SPI_SPEED    (SPI_CLK_1MHZ)
 #define SX126X_SPI_MODE     (SPI_MODE_0)
 
+#ifndef SX126X_PERIPH_WAKEUP_TIME
+#if defined(CPU_FAM_STM32WL)
+#define SX126X_PERIPH_WAKEUP_TIME   1000U
```

Comment:
why there are 2 wake up times? Shouldn't this be defined in the radio?

Reply:
The STM32WL [Application note](https://www.st.com/resource/en/application_note/dm00660451-how-to-build-a-lora-application-with-stm32cubewl-stmicroelectronics.pdf) Section 4.3 states RF wakeup time as 10ms.

Reply:
then this is still wrong, since 1000 would map to 1 ms.

Reply:
> then this is still wrong, since 1000 would map to 1 ms.

Already updated.